src/test/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/MarkDuplicatesDataflowIntegrationTest.java

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.hellbender.tools.dataflow.pipelines;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.tools.picard.sam.markduplicates.MarkDuplicatesIntegrationTest;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.text.XReadLines;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Objects;
+
+
+public class MarkDuplicatesDataflowIntegrationTest extends CommandLineProgramTest{
+
+    @DataProvider(name = "md")
+    public Object[][] md(){
+        return new Object[][]{
+                {new File(MarkDuplicatesIntegrationTest.TEST_DATA_DIR,"example.chr1.1-1K.unmarkedDups.noDups.bam"), 20, 0},
+                {new File(MarkDuplicatesIntegrationTest.TEST_DATA_DIR,"example.chr1.1-1K.unmarkedDups.bam"), 90, 6},
+                {new File(MarkDuplicatesIntegrationTest.TEST_DATA_DIR,"example.chr1.1-1K.markedDups.bam"), 90, 6},  //90 total reads, 6 dups
+        };
+    }
+
+    @Test(groups = "dataflow", dataProvider = "md")
+    public void testMarkDuplicatesDataflowIntegrationTestLocal(final File input, final long totalExpected, final long dupsExpected) throws IOException {
```

Comment:
You can look at BaseRecalibratorDataflowIntegrationTest, ApplyBQSRDataflowIntegrationTest, SmallBamWriterTest.
We use the "bucket" group to mark tests that read from GCS, and "cloud" group to mark those that execute code on the cloud.